src/js/components/TaskTable.js

```diff
@@ -39,9 +39,59 @@ class TaskTable extends React.Component {
   }
 
   getStatusValue(task) {
+    let taskHealth = this.getTaskHealth(task);
+    if (taskHealth === true) {
+      return 'Healthy';
+    }
+    if (taskHealth === false) {
+      return 'Unhealthy';
+    }
+
     return TaskStates[task.state].displayName;
   }
 
+  getTaskHealth(task) {
+    let mesosTaskHealth = this.getTaskHealthFromMesos(task);
+    if (mesosTaskHealth !== null) {
+      return mesosTaskHealth;
+    }
+    let marathonTaskHealth = this.getTaskHealthFromMarathon(task);
```

Comment:
Couldn't we just return `this.getTaskHealthFromMarathon(task)` here, since the fallback is returning `null`